.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
I think we should keep this out of the repo.

Reply:
Just removed it, should be ok now!